meetings/2022-05-12.md

```diff
@@ -0,0 +1,32 @@
+# ISIS TC Meeting May 12, 2022 @ 1PM PDT & MST / 4PM EDT
+
+## [Teams Link](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2f_%23%2fl%2fmeetup-join%2f19%3ameeting_YWRkZjdiMGUtZWJlOC00OWMzLThlMTItZTk0Y2MyM2E1MWE0%40thread.v2%2f0%3fcontext%3d%257b%2522Tid%2522%253a%25220693b5ba-4b18-4d7b-9341-f32f400a5494%2522%252c%2522Oid%2522%253a%2522c27c6e98-e45a-45ff-aea5-7f10d6fe67c1%2522%257d%26anon%3dtrue&type=meetup-join&deeplinkId=e54b3969-3c7f-4efb-9cad-ee99cf639f86&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true)
+
+### Attending
+
+-
+
+## Agenda / Notes
+
+- Action Items from last month
+  - Jesse: Update contributing documents from experience with #4887
+  - Kris: Post enhancement request to bundle IAKs with ISIS similar to appdata
+  - Ross: Reply to open discussion on API PR
+
+- API definition PR [#4792](https://github.com/USGS-Astrogeology/ISIS3/pull/4792)
+- ISIS name change WG update
+
+- Open discussion
+
+
+## Discussions for next meeting
+
+- ISIS Name Change Proposal
```

Comment:
I went ahead and added this to the for next meeting agenda so we don't lose track of it. I'd like to set aside a fair amount of time for the discussion in June.